core/capabilities.go

```diff
@@ -341,11 +341,16 @@ func (bcast *Capabilities) LivepeerVersionCompatibleWith(orch *net.Capabilities)
 		return false
 	}
 
-	// Ignore prerelease versions as in go-livepeer we actually define post-release suffixes
-	minVerNoSuffix, _ := minVer.SetPrerelease("")
-	verNoSuffix, _ := ver.SetPrerelease("")
+	// If minVersion has no pre-release component, ignore pre-release versions by
+	// default as in go-livepeer, we define post-release suffixes.
```

Comment:
@victorges I noticed that I typically use punctuation in my comments, whereas it's not common practice here. I'm happy to adjust and remove it if preferred.